platforms/common/include/px4_platform_common/atomic.h

```diff
@@ -66,11 +70,11 @@ class atomic
 {
 public:
 
-#ifdef __PX4_NUTTX
+#if defined(__PX4_POSIX)
```

Comment:
```suggestion
#if !defined(__PX4_NUTTX)
```

Reply:
It's also skipped on QuRT where the `_value` is simply volatile.

Reply:
Alright. I slightly prefer the explicit exclusion list to avoid surprises.